cv-image-to-xml/src/main/java/org/appdevforall/codeonthego/computervision/ui/viewmodel/ComputerVisionViewModel.kt

```
@@ -139,32 +150,20 @@ class ComputerVisionViewModel(
         }
 
         return try {
             val rotatedBitmap = Bitmap.createBitmap(
                 bitmap, 0, 0, bitmap.width, bitmap.height, matrix, true
             )
-            // Clean up original if a new one was created
             if (rotatedBitmap != bitmap) {
                 bitmap.recycle()
             }
             rotatedBitmap
         } catch (e: OutOfMemoryError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)